### PR TITLE
feat(collection): update find methods

### PR DESCRIPTION
**Pull request by Gio85**

### CODE REVIEW

**Review by useless-stuff — APPROVED** [review]

**Review by Dextorr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,4 +128,78 @@ async function initMyService(): Promise<any> {
...
+    firstName: 'mario',
+    lastName: 'bros',
+    email: 'mario@bros.com',
+    createdAt: new Date()
+}
+await usersCollection.createDocument(userData)
+
+// Find document
+const user = await usersCollection.findDocument({ email: 'mario@bros.com' })
+// user => { firstName: 'mario', lastName: 'bros', id: '1234-1234-1234-1234', email: 'mario@bros.com', cretedAt: '2021-02-05T14:42:44.758Z' }
```
Comment (julian-mp) [inline README.md:163]: createdAt

File: README.md [hunk 2]
```diff
@@ -127,4 +128,78 @@ async function initMyService(): Promise<any> {
...
+
+// Find documents
+const users = await usersCollection.findDocuments({ email: 'mario@bros.com' })
+// users => { data: [ { firstName: 'mario', lastName: 'bros', id: '1234-1234-1234-1234', email: 'mario@bros.com', cretedAt: '2021-02-05T14:42:44.758Z' } ] }
+```
+
```
Comment (julian-mp) [inline README.md:169]: same above

File: README.md (hunk 1 above)
Comment (Gio85) [reply README.md:163]: done

File: README.md (hunk 2 above)
Comment (Gio85) [reply README.md:169]: done

**Review by julian-mp — APPROVED** [review]

**Pull request merged**